content/member-handbook.md

```diff
@@ -147,6 +147,137 @@ That said, here are some things we might do, in rough order of severity:
   Usually used when someone refuses to step back from a heated conversation or for trolls.
 - **ban**:  asking someone to log off completely or doing the same via admin tools.
 
+## Job Postings
+
+The `#jobs` channel is a space that we've dedicated for people to post when they
+are looking for work, or when they have jobs.  This section will provide some
+guidance and rules for posting in the jobs channel.
+
+### When You're Looking for Work
+
+The `#jobs` channel is available for anyone to use if they are looking for
+technical work.  If you're looking for work, feel free to post a link to your
+resume, linkedin profile, github page, etc.  You may also pin a message saying
+that you are looking for work, provided you unpin when you are no longer on the
+market.
+
+You should not broadcast your availability more than once a week.
+
+### When You're Hiring
+
+Posting in the `#jobs` channel is a great way to find local talent, whether
+you're an individual contributor looking for a new teammate, a hiring manager
+looking to fill a role, or an internal or external recruiter, you're welcome to
+post about your job openings so long as you adhere to our guidelines and rules.
+
+#### Job Location
+
+The STL Tech Slack is concerned with helping people who are in St. Louis find
+jobs.  To that extent, we ask that all jobs that are posted be jobs that can be
+carried out by people in St. Louis.  This means local on-sight, local remote,
+and full-time remote jobs are all okay.  There are no strict geographical bounds
```

Comment:
What's the difference between local remote vs full-time remote?

Reply:
I was using "local remote" for things where the company is in St. Louis and you can commute to the office, but can typically work from home.  I'll add a clarification